infrastructure/template/root/.github/workflows/test-and-pre-release.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       matrix:
         # Remove unsupported versions and add more versions. Use LTR version in the cov_tests job
-        docker_tags: [release-3_10, latest]
+        docker_tags: [release-3_16, latest]
```

Comment:
3.10 is still marked as a minimum version in metadata.txt. Basically the API is still compatible with QGIS 3.10, so maybe we don't want to drop it off the template just yet. (In the future, sure!)

Reply:
Ah okay, will revert this!